cmd/skupper/skupper_podman_site.go

```diff
@@ -78,7 +80,35 @@ func (s *SkupperPodmanSite) Create(cmd *cobra.Command, args []string) error {
 	if routerCreateOpts.Ingress != types.IngressNoneString {
 		// Validating ingress hosts (required as certificates must have valid hosts)
 		if len(site.IngressHosts) == 0 {
-			return fmt.Errorf("At least one ingress host is required")
+			var ingressHosts []string
+			// Get hostname of the machine
+			hostname, err := os.Hostname()
+			if err == nil {
+				ingressHosts = append(ingressHosts, hostname)
+			} else {
```

Comment:
This part should not be in the else. It's better to always run it, as it
would use all IP4 addresses as part of the valid set of ingress-hosts.